src/cleo/commands/completions/templates.py

```diff
@@ -122,4 +122,39 @@
 %(cmds_opts)s"""
 
 
-TEMPLATES = {"bash": BASH_TEMPLATE, "zsh": ZSH_TEMPLATE, "fish": FISH_TEMPLATE}
+POWERSHELL_TEMPLATE = (
+    """\
+$%(function)s = {
+    param(
+        [string] $wordToComplete,
+        [System.Management.Automation.Language.Ast] $commandAst,
+        [int] $cursorPosition
+    )
+
+    $options = %(opts)s
+    $commands = %(cmds)s
+
+    if ($wordToComplete -notlike '--*' -and $wordToComplete -notlike "" -and """
```

Comment:
`poetry ` then <kbd>alt</kbd>+<kbd>tab</kbd> doesn't show the commands, only the options:

![image](https://github.com/python-poetry/cleo/assets/65446343/d305e9c9-7b0a-4e79-8c8a-0c52a558c8bf)